apps/penxle.com/src/lib/tiptap/node-views/access-barrier/Component.svelte

```diff
@@ -40,74 +50,150 @@
     }
   `);
 
-  const handlePriceChange: FormEventHandler<HTMLInputElement> = (event) => {
-    const price = Number(event.currentTarget.value.replaceAll(',', ''));
-    if (Number.isNaN(price)) {
-      return;
-    }
+  const [floatingRef, floatingContent, update] = createFloatingActions({
+    strategy: 'absolute',
+    placement: 'bottom',
+    middleware: [offset(8)],
+  });
+
+  const { form, errors, data, setFields, setInitialValues } = createForm({
+    initialValues: { price: 0 },
+    extend: [
+      validator({
+        schema: z.object({
+          price: z.number().int().min(100).max(1_000_000).multipleOf(100),
+        }),
+      }),
+    ],
+    onSubmit: (values) => {
+      updateAttributes({ price: values.price });
+      priceOpen = false;
+    },
+  });
 
-    updateAttributes?.({ price });
-  };
+  $: if (priceOpen) {
+    // eslint-disable-next-line unicorn/prefer-top-level-await
+    void tick().then(() => {
+      update();
+      priceInputEl?.focus();
+    });
+  }
+
+  afterNavigate(() => {
+    priceOpen = false;
+  });
 </script>
 
 {#if editor?.isEditable}
-  <NodeView class="my-4 flex center gap-4 border border-secondary rounded-xl p-4" data-drag-handle>
-    <div class="flex grow gap-4">
-      <i class="i-lc-circle-dollar-sign square-6" />
-
-      <div class="flex grow flex-col">
-        <div class="body-16-b">결제 상자를 옮겨 유료분량을 만들어 보세요</div>
-        <div class="body-13-m text-secondary mt-1">이 상자의 이후 지점부터 결제를 통해 감상할 수 있어요</div>
-
-        <div class="flex gap-2 items-center mt-4">
-          <TextInput
-            class="bg-primary pl-4 py-2 body-16-b rounded-lg"
-            inputmode="numeric"
-            value={price?.toLocaleString()}
-            on:input={handlePriceChange}
-          >
-            <span slot="right-label" class="pr-4 bodylong-16-m text-disabled">포인트</span>
-          </TextInput>
-        </div>
-      </div>
+  <NodeView
+    class={clsx('flex justify-end relative py-8px', selected && 'ring-2 ring-teal-500 rounded-6px')}
```

Comment:
```suggestion
    class="flex justify-end relative py-8px aria-selected:(ring-2 ring-teal-500 rounded-6px)"
    aria-selected={selected}
```

P3: aria state 로 스타일링할 수 있는 상황에는 적용하는 것이 스타일 코드가 더 간결해진다고 생각합니다.